presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestGrantRevoke.java

```diff
@@ -128,41 +135,29 @@ public void testPublic()
     @Test(groups = {AUTHORIZATION, HIVE_CONNECTOR, PROFILE_SPECIFIC_TESTS})
     public void testTableOwnerPrivileges()
     {
-        String tableName = "alice_table";
-        QueryExecutor aliceExecutor = connectToPresto("alice@presto");
-
-        aliceExecutor.executeQuery(format("DROP TABLE IF EXISTS %s", tableName));
-        aliceExecutor.executeQuery(format("CREATE TABLE %s(month bigint, day bigint)", tableName));
-
         onHive().executeQuery("set role admin;");
-        assertThat(onHive().executeQuery(format("SHOW GRANT USER alice ON TABLE %s", tableName)).
-                project(1, 2, 5, 6, 7, 8, 10)).containsOnly(// Select only the relevant columns of SHOW GRANT
+        assertThat(onHive().executeQuery(format("SHOW GRANT USER %s ON TABLE %s", PRINCIPAL_NAME, tableName)).
+                project(1, 2, 5, 6, 7, 8, 10)).containsOnly(// Skip unknown columns such as "grant_time" of SHOW GRANT
```

Comment:
From my perspective, looking at the code, I have no clue what SHOW GRANTS from Hive returns. So I don't know what it should be if the test starts failing for some reason. It's much easier to put that in a comment rather than having to look up the syntax for Hive every time. If you want you can add the columns we exclude too :)